core/templates/dev/head/pages/exploration_editor/editor_tab/state_editor.html

```diff
@@ -25,7 +25,8 @@
     <div ng-if="areFallbacksEnabled()">
       {% include 'pages/exploration_editor/editor_tab/state_editor_fallbacks.html' %}
     </div>
-    <div>
+    <!-- Hint Editor UI will be enabled once the entire pipeline is built. -->
+    <div ng-hide="true">
```

Comment:
Oh, that's not so great. Can you pull this boolean out into a constant in app.js or feconf, similar to what we do for e.g. SHOW_TRAINABLE_ANSWERS (for the ML training interface)? It should be easy for devs to turn it on and off; it's a bit hidden here.

Thanks!